closure/goog/ui/tree/basenode.js

```diff
@@ -186,16 +186,16 @@ goog.ui.tree.BaseNode.prototype.initAccessibility = function() {
     }
 
     var img = this.getIconElement();
-    if (img) {
+    if (img && img.setAttribute) {
```

Comment:
What edge case is this intended to address?  Shouldn't the return type of getIconElement already account for it, or is there something missing?